packages/panzer/disc-fe/src/Panzer_SubcellConnectivity.hpp

```diff
@@ -69,15 +69,17 @@ class SubcellConnectivity
    * \return Number of subcells associated with the cells
    */
   KOKKOS_INLINE_FUNCTION
-  int numSubcells() const {return _num_subcells;}
+  int numSubcells() const {return _num_subcells_device(0);}
+  int numSubcellsHost() const {return _num_subcells;}
```

Comment:
I think (double check) you can just return `subcell_to_cells_adj.extent(0)-1`. That would mean we don't need a host version of the call.

Reply:
done